src/LocalizationEditor/__snapshots__/LocalizationEditor.test.js.snap

```diff
@@ -22,8 +22,12 @@ exports[`<LocalizationEditor /> create localization editor 1`] = `
         className="fd-popover__control"
       >
         <div
+          aria-haspopup={true}
           className="fd-input-group fd-input-group--after"
           onClick={[Function]}
+          onKeyPress={[Function]}
+          role="button"
+          tabIndex={0}
         >
```

Comment:
Similar behavior as `ComboboxInput` where `space` key presses are consumed to open the popup.